core/App/components/chat/ChatEvent.tsx

```diff
@@ -1,22 +1,21 @@
 import React from 'react'
 import { useTranslation } from 'react-i18next'
-import { Text, View } from 'react-native'
-
-import { useTheme } from '../../contexts/theme'
+import { Text, TextStyle, View } from 'react-native'
 
 interface ChatEventProps {
   userLabel?: string
   actionLabel?: string
+  userTextStyle: TextStyle
```

Comment:
I suggest passing `enum Position { Left , Rigt }` property instead of styles objects, and define styles inside ChatEvent component.